src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx

```diff
@@ -79,12 +78,12 @@ const Signup = ({signUpRequest, studentSignUpData}) => {
   );
 };
 
-Signup.propTypes = {
-  signUpRequest: PropTypes.func.isRequired,
-  studentSignUpData: PropTypes.shape({
-    isFetching: PropTypes.bool,
-    success: PropTypes.bool,
-    msg: PropTypes.string,
+Signup = {
+  signUpRequest: func.isRequired,
+  studentSignUpData: shape({
+    isFetching: Boolean,
+    success: Boolean,
+    msg: String,
   }).isRequired,
```

Comment:
same here.

Reply:
in this i have removed

Reply:
did i do something wrong?

Reply:
Yes You have removed the prop-types here.

```
Signup = {
  signUpRequest: func.isRequired,
  studentSignUpData: shape({
    isFetching: Boolean,
    success: Boolean,
    msg: String,
  }).isRequired,
};
```